File: epi_judge_cpp/is_valid_parenthesization.cc
#include <string>

using std::string;

bool IsWellFormed(const string& s) {
  // Implement this placeholder.
  return true;
}

#include "test_framework/generic_test.h"

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"s"};
  GenericTestMain(args, "is_valid_parenthesization.tsv", &IsWellFormed,
                  DefaultComparator{}, param_names);
  return 0;
}

File: epi_judge_java/epi/IsValidParenthesization.java
package epi;

import epi.test_framework.EpiTest;
import epi.test_framework.GenericTest;

public class IsValidParenthesization {
  @EpiTest(testfile = "is_valid_parenthesization.tsv")

  public static boolean isWellFormed(String s) {
    // Implement this placeholder.
    return true;
  }

  public static void main(String[] args) {
    System.exit(GenericTest
                    .runFromAnnotations(
                        args, new Object() {}.getClass().getEnclosingClass())
                    .ordinal());
  }
}
